modules/services/rosetta2-gc/default.nix

```diff
@@ -0,0 +1,61 @@
+{
+  config,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkEnableOption
+    mkIf
+    mkOption
+    ;
+
+  launchdTypes = import ../../launchd/types.nix { inherit config lib; };
+
+  logFile = "/var/log/rosetta2-gc.log";
+
+  cfg = config.rosetta2-gc;
+in
+
+{
+  options.rosetta2-gc = {
```

Comment:
```suggestion
  cfg = config.services.rosetta2-gc;
in

{
  options.services.rosetta2-gc = {
```

Reply:
Maybe it would be better to move this under `virtualisation.rosetta.gc.*` to be more similar to NixOS

Reply:
Can you move the file to a virtualisation folder?

Reply:
Updated. Also made the name say rosetta2, since what we're doing might be specific to that.

Reply:
Rosetta is preferable as Rosetta 1 refers to the software used to migrate from PowerPC to Intel Macs released in 2006.

Reply:
Alright! Updated.